types/index.d.ts

```diff
@@ -0,0 +1,141 @@
+// TypeScript Version: 3.0
+
+import {Node} from 'unist'
+import stringifyEntities = require('stringify-entities')
+
+declare namespace hastUtilToHtml {
+  interface HastUtilToHtmlOptions {
+    /**
+     * Whether the *root* of the *tree* is in the `'html'` or `'svg'` space (enum, `'svg'` or `'html'`, default: `'html'`).
```

Comment:
is the `enum` part needed? since this is described by the typing?
would it help to annotate the default with tsdoc `@default` or `@defaultValue`?

Reply:
the same goes for the other type/default doc comments

Reply:
I don't have any preferences about this unless we want to generate documentation from the doc comments.

Reply:
I lean toward using TSDoc comments, the typescript language server (used by VSCode and other editors), can provide hints and documentation on hover based off annotated comments, which it can't do quite as well without the annotations.

Reply:
If so, we could give a shot. First of all, I think we should discuss how are we going to use TSDoc stuff. Could you submit another pr based on this branch? I think it would be much faster for us to determine the strategy because I don't have any strong opinion yet.